main-command/src/main/scala/sbt/internal/LabeledFunctions.scala

```diff
@@ -0,0 +1,82 @@
+/*
+ * sbt
+ * Copyright 2011 - 2018, Lightbend, Inc.
+ * Copyright 2008 - 2010, Mark Harrah
+ * Licensed under Apache License 2.0 (see LICENSE)
+ */
+
+package sbt.internal
+
+private[sbt] object LabeledFunctions {
```

Comment:
Could you write up the motivation for this change (what you wrote in PR description) in both commit message and Scaladoc here plz?

Reply:
I noticed that the commit message is already there, so just the Scaladocs.